schema-lending.graphql

```diff
@@ -323,6 +608,9 @@ type UsageMetricsDailySnapshot @entity {
   " ID is # of days since Unix epoch time "
   id: ID!
```

Comment:
should we make everything `Bytes` for consistency (and perf improvement)?

Reply:
This is what happens when you try to do something like this: 

```
  " { Market Address }-{ Input Token Address }-{ Optional: Snapshot ID } "
  id: Bytes!
```
<img width="617" alt="Screen Shot 2022-11-17 at 2 35 16 AM" src="https://user-images.githubusercontent.com/60583747/202364963-62df6b72-fcad-46a3-a591-b978344e44b4.png">

Reply:
Just kidding, I was able to get it to work by converting the i32 > Address > Bytes

Reply:
There are some entities that have words in them though, so those will have to remain ID